packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -157,6 +157,33 @@ void main() {
     );
   });
 
+  testWidgets(
+      'Blur affect is disabled when disableBackgroundFilterBlur is true', (WidgetTester test) async {
+    await test.pumpWidget(
+      const CupertinoApp(
+        theme:  CupertinoThemeData(brightness: Brightness.light),
```

Comment:
Nit: here and below, there seem to be two extra spaces after the `:`.

Reply:
Thanks for your feedback made the required changes.